src/cmd/cli/command/commands.go

```diff
@@ -56,6 +56,27 @@ func prettyError(err error) error {
 	return err
 }
 
+func interactiveComposeDown(ctx context.Context, c cliClient.Client, projectName string) error {
```

Comment:
Shall we move this function to `composeDown.go`? This file is pretty large already

Reply:
Ok, done! There are a few adjustments I made to my previous code to make it work (i.e. the hints are now done by the caller)